recipes/gitsome/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set version = "0.5.0" %}
+
+package:
+  name: gitsome
+  version: {{ version }}
+
+source:
+  fn: gitsome-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/g/gitsome/gitsome-{{ version }}.tar.gz
+  md5: 85217683d7c4c08fd2a9d3b415c640c9
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - gh = gitsome.main_cli:cli
+
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpydoc >=0.5
+    - ply >=3.4
+    - prompt_toolkit >=0.52
```

Comment:
Yeah, I agree.  When I asked @jonathanslenders about this,  he more or less said that ptk <1.0 is now unsupported.  We have likewise dropped support for earlier versions in xonsh.